tests/templates/test_subroutines/test_qpe.py

```diff
@@ -136,6 +144,143 @@ def test_phase_estimated_two_qubit(self):
         # lower
         assert np.allclose(estimates[-1], phase, rtol=1e-2)
 
+    @pytest.mark.parametrize("param", np.linspace(0, 2 * np.pi, 4))
+    def test_phase_estimated_single_ops(self, param):
+        """Tests that the QPE works correctly for a single operator"""
+
+        unitary = qml.RX(param, wires=[0])
+
+        # Analytical eigenvectors and phase of the unitary
+        eig_vec = np.array([-1 / np.sqrt(2), 1 / np.sqrt(2)])
+        phase = param / (4 * np.pi)
+
+        estimates = []
+        wire_range = range(3, 11)
+
+        for wires in wire_range:
+            dev = qml.device("default.qubit", wires=wires)
+
+            estimation_wires = range(1, wires - 1)
+            target_wires = [0]
+
+            with qml.tape.QuantumTape() as tape:
+
+                # Prepare the eigenstate of the circuit
+                qml.QubitStateVector(eig_vec, wires=target_wires)
+
+                # Perform QPE, using default target wires defined in the operator
+                qml.QuantumPhaseEstimation(unitary, estimation_wires=estimation_wires)
+
+                # Find probabilities
+                qml.probs(estimation_wires)
```

Comment:
```suggestion
            tape = qml.tape.QuantumScript([qml.QuantumPhaseEstimation(unitary, estimation_wires=estimation_wires)],
                [qml.probs(estimation_wires)], prep = [ qml.QubitStateVector(eig_vec, wires=target_wires)])
```
We're trying to move away from using `QuantumTape` internally.  Not deprecated yet, but this is now the recommended way of doing things.